### DISCUSSION TITLE
Cameras that work with Linux Motion

### DISCUSSION THREAD

**Discussion by Thorium9O** [Motion-Project/motion#1667] [Q&A]
Does anyone know if there is a list of cameras out there that are known to work with Linux Motion? It seems simple at first but I have been struggling to get a Swann camera to connect and from what I'm reading it might be impossible for some reason. I'm looking to connect the cameras over an ethernet cable. 

Thanks in advance,

**Comment by Mr-Dave** [comment]
Make sure that somewhere in the specifications the camera lists providing a rtsp or http stream.  If the camera/company provides a "monthly service"  it is virtually guaranteed to not work with anything except their service.  Swann may be mixed on which cameras provide a rtsp feed or not.  You may want to use internet searches to see whether anyone has posted a working rtsp connection string for the camera you have.
